lib/yargs-factory.ts

```diff
@@ -1398,7 +1398,7 @@ export class YargsInstance {
     this.describe(this.#versionOpt, msg);
     return this;
   }
-  wrap(cols: number | null | undefined): YargsInstance {
+  wrap(cols: number | null | undefined = this.terminalWidth()): YargsInstance {
```

Comment:
@seivan will `this.terminalWidth()` only be invoked if an `undefined` value is provided?

wondering if we should instead write as:

```js
wrap(cols: number | null | undefined): YargsInstance {
  if (cols === undefined) cols = this.terminalWidth()
}
```

Reply:
@bcoe That depends. 
Do you think "null" is a valid option, in which case the current implementation is better. 
```ts
wrap(cols: number | null | undefined = this.terminalWidth())
```

Otherwise, if null is never a valid option, I'd suggest to switch to 
```ts
wrap(cols = this.terminalWidth())
```